airbyte-integrations/connectors/destination-databend/integration_tests/integration_test.py

```diff
@@ -0,0 +1,159 @@
+#
```

Comment:
Have you considered relying on the destination acceptance tests as a baseline for correctness of your connector? They test a lot more scenarios than covered here. They will require some Java coding on your side though.

Do the current tests assume there is a running Databend instance to connect to? If so I suggest running a databend docker container as part of the tests https://github.com/testcontainers/testcontainers-python

Reply:
Ok, I will consider the destination acceptance tests as a baseline and would you like to give me some docs about writing the tests code? Or I can add the test cases later because I have already successfully run the airbyte + dbt normalization pipeline.
![image](https://user-images.githubusercontent.com/7600925/208065401-b1e52d1a-e39c-4c3a-bb5d-e049fa34adb6.png)

Another question, the current test requires a connection string to connect to databend cloud. So I don't konw how to provide it in tests and I wonder how snowflake mock there client and I will do a survey on it🤔.